### PR TITLE
Make elements keyboard accessible

### PR DESCRIPTION
**Pull request by geramirez**
**This is a work in progress**, I just wanted to get some feedback if the changes are OK. Then I will add tests and clean up the code.

### CODE REVIEW

**Comment by paranoidjk** [comment]
LGTM.  but should the tabIndex be different？

**Comment by geramirez** [comment]
Ok, we've made the final changes. As for tests, it looks like they were failing with version 6.x of rc-tools, but will pass with 5.x. We did not revert the version in this commit, but that might be necessary for the tests to pass.

If you're interested we can update the tests to use the [enzyme](https://github.com/airbnb/enzyme) library. Let us know.

**Comment by paranoidjk** [comment]
rc-tools 6.x have moved all test command to a new package called rc-test，you can just use rc-test to replace the relative test command，then ci should be no problem any more.


But if you are willing to refactor test with jest and enzyme，that would be great too.  You can see https://github.com/react-component/slider test code as a example.

**Comment by geramirez** [comment]
@paranoidjk it appears that CI was broken before this PR. We checked that we did not create new errors by running the tests before and after this feature update. 

Would this be ok to merge at this point?

File: src/Pagination.jsx [hunk 1]
```diff
@@ -190,6 +190,12 @@ class Pagination extends React.Component {
     return this.state.current < this._calcPage();
   }
 
+  _runIfEnter(event, callback) {
+    if (event.keyCode === 'Enter') {
```
Comment (paranoidjk) [inline src/Pagination.jsx]: You should use `event .key === 'Enter'` or `event.charCode  === 13`, see https://facebook.github.io/react/docs/events.html#keyboard-events

**Comment by paranoidjk** [comment]
@geramirez As i mentioned earlier, just follow this https://github.com/react-component/checkbox/commit/cd7c340d8bbf52e62da61b1ea6ee80787d21d9d8 to fix the ci.

File: src/Pagination.jsx (hunk 1 above)
Comment (geramirez) [inline src/Pagination.jsx]: Ok, the update should be ready.

**Comment by geramirez** [comment]
Ok, @paranoidjk the build is fixed. 
We also added [rc-test](https://github.com/react-component/pagination/pull/83/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R49) and [fixed the test that was broken](https://github.com/react-component/pagination/pull/83/files#diff-68eac779f6d6e65e0110ebd09647b059L188)

**Pull request merged**